apps/projects/templates/meinberlin_projects/includes/project_list_tile.html

```diff
@@ -3,8 +3,9 @@
 <div class="tile">
     <div class="tile__head">
         <a tabindex="-1" class="tile__image tile__image--shadowed"
+           {% if project.image %}style="background-image: url({{ project.image|thumbnail_url:'project_thumbnail' }})"{% endif %}
            href="{% extproject_url project 'project-detail' %}" target="{{ project|is_external|yesno:"_blank,_self" }}">
-            <img src="http://placehold.it/285x170" alt="">
+            <span class="sr-only">{{ project.name }}</span>
```

Comment:
You are right. In that case I would opt for `aria-hidden="true"`. The image does not add anything other than the image itself, so it can just as well be ignored.